unblob/pool.py

```diff
@@ -38,10 +40,12 @@ def is_empty(self) -> bool:
             return self._unfinished_tasks._semlock._is_zero()  # type: ignore
 
 
+_SENTINEL = ...
```

Comment:
It's quite unusual to use Ellipsis for a sentinel object, because it's a singleton object like True and False. It also indicates unfinished code, for example in a function body.
A more common way is to make a unique value not used anywhere else in the codebase, which can be simply achieved by instantiating an empty object.
```
>>> a = ...
>>> b = ...
>>> a is b
True
>>> c = object()
>>> d = object()
>>> c is d
False
>>>
```

Reply:
Neat!  I will fix this in https://github.com/IoT-Inspector/unblob/pull/227, I need to fix some issue in the Pool anyways.